cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -20,11 +20,12 @@ desiredState:
         dnsPolicy: "Default"  # Don't use cluster DNS.
         containers:
           - name: etcd
-            image: quay.io/coreos/etcd:latest
+            image: quay.io/coreos/etcd:v2.0.3
             command: [
-                    "/etcd",
-                    "-bind-addr=127.0.0.1",
-                    "-peer-bind-addr=127.0.0.1",
+                    # entrypoint = "/etcd",
```

Comment:
Does this work? The command seems wrong to me.

Also can you stick to the existing format for yaml files in other examples. Example:
```shell
command:
          - "/bin/sh"
          - "-c"
          - "echo ok
```